src/Algebra/Properties/Loop.agda

```diff
@@ -0,0 +1,41 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some basic properties of Quasigroup
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Algebra.Bundles using (Loop)
+
+module Algebra.Properties.Loop {l₁ l₂} (L : Loop l₁ l₂) where
+
+open Loop L
+open import Algebra.Definitions _≈_
+open import Relation.Binary.Reasoning.Setoid setoid
+open import Data.Product
+open import Algebra.Properties.Quasigroup
+
+x/x≈ε : ∀ x → x // x ≈ ε
+x/x≈ε x = begin
```

Comment:
Hmm should we be using the actual operations `//` in the names rather than `/`?

Reply:
Not sure. I like to have `/`. In some places we don't explicitly mention `∙` too. But if you insist I will change it. Let me know!! :)

Reply:
@MatthewDaggitt Sorry I am waiting for your comments. (Or am I missing something?)

Reply:
Apologies @Akshobhya1234 this completely slipped off my radar. Yes, I think its best to use the correct operator name in the proof names. I think it's different from `∙` because with that we're simply omitting the name, but here you're _changing_ the name...

Reply:
Done!